js/app/application.js

```diff
@@ -43,6 +43,19 @@ const KnowledgeSearchIface = '\
   </interface> \
 </node>';
 
+const GrandCentralContentIface = '\
+<node> \
+  <interface name="com.endlessm.GrandCentralContent"> \
+    <method name="ContentQuery"> \
```

Comment:
Are you sure this particular interface should have a query parameter? As we discussed last week, I think that we should use different DBus interfaces for different content capabilities, and the most basic one should probably take no arguments.